src/modules/navigator/mission_feasibility_checker.cpp

```diff
@@ -124,13 +125,14 @@ MissionFeasibilityChecker::checkRotarywing(dm_item_t dm_current, size_t nMission
 
 		// look for a takeoff waypoint
 		if (missionitem.nav_cmd == NAV_CMD_TAKEOFF) {
-			// make sure that the altitude of the waypoint is at least one meter larger than the acceptance radius
+			// make sure that the altitude of the waypoint is at least one meter larger than the altitude acceptance radius
 			// this makes sure that the takeoff waypoint is not reached before we are at least one meter in the air
 			float takeoff_alt = missionitem.altitude_is_relative ? missionitem.altitude : missionitem.altitude - home_alt;
 
 			// check if we should use default acceptance radius
-			float acceptance_radius = default_acceptance_rad;
+			float acceptance_radius = default_altitude_acceptance_rad;
 
+            // if a specific acceptance radius has been defined, use that one instead
```

Comment:
Tabs instead of spaces. Usually CI complains about this stuff, but I guess it's lenient on comments.